zerver/lib/events.py

```diff
@@ -1636,6 +1641,7 @@ def do_events_register(
         pronouns_field_type_supported=pronouns_field_type_supported,
         linkifier_url_template=linkifier_url_template,
         user_list_incomplete=user_list_incomplete,
+        archived_streams=archived_streams,
```

Comment:
Doesn't this need to be passed to `request_event_queue` too?

Reply:
I mean it is harmless to pass `archived_stream` parameter but I don't see the point. Like what will that do?
Anyway I updated the code according to you.